core/controllers/dashboard.py

```diff
@@ -148,9 +148,9 @@ def _get_intro_card_color(category):
             subscription_services.get_exploration_ids_subscribed_to(
                 self.user_id))
 
-        subscribed_exploration_summaries = (
+        subscribed_exploration_summaries = filter(None, (
             exp_services.get_exploration_summaries_matching_ids(
-                exploration_ids_subscribed_to))
+                exploration_ids_subscribed_to)))
         subscribed_collection_summaries = (
```

Comment:
For collections, it's not *needed* . But IMO it would be better if we use it in a similar manner rather than skipping any collection_summary which is None inside the loop:  https://github.com/oppia/oppia/blob/develop/core/controllers/dashboard.py#L178
This will also make the code more consistent to read. What do you think ?